shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -250,7 +253,18 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         }
 
         result.setSelected(object.hasFlag(Flag.IS_SELECTED));
-        result.setText(object.getValueLabelHint());
+        if (object.hasFlag(Flag.IS_TEXT_FIELD)) {
+            if (hintTextProxy.canSetHintText()) {
+                result.setText(object.getValue());
+                hintTextProxy.setHintText(result, object.getLabel());
+            } else if (object.hasFlag(Flag.IS_FOCUSED)) {
+                result.setText(object.getValue());
+            } else {
+                result.setText(object.getLabel());
```

Comment:
Now that I see this: It's odd that you can't access what you types into a text field until it's focused. Not sure how to implement that with the limited Android API though...

Reply:
I'll revert this for now until I find a better solution